spread.yaml

```diff
@@ -57,6 +57,13 @@ backends:
           storage: 12G
           workers: 6
 
+      - fedora-39-64:
```

Comment:
If your intention is to run the remote-build spread tests on these platforms, I think you need to add them [here](https://github.com/snapcore/snapcraft/blob/8eef63ca396eda0d4670b905cb1f7afb170baa09/.github/workflows/spread-scheduled.yaml#L78)

Looking at the test, I realize there was no need to define platforms [here](https://github.com/snapcore/snapcraft/blob/8eef63ca396eda0d4670b905cb1f7afb170baa09/tests/spread/general/remote-build/task.yaml#L4) since it's a manual test.

Reply:
that's a removal of environments though, I thought it would pick up just fine :shrug: 

Perhaps I need to remove fedora from everything else though

Reply:
I removed the fedora test for now, will add it in a new PR